msgpack_test.go

```diff
@@ -1,4 +1,4 @@
-package msgpack_test
+package msgpack
```

Comment:
To allow to run these tests outside of 'official' tree i.e. 'gopkg.in/vmihailenco/msgpack.v2'.
This helps to contributors to fork and test your own code.